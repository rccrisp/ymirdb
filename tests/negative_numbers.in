set a -1 -5 3
min a
max a
len a
sum a
bye