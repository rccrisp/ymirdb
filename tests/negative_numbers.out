> ok

> -5

> 3

> 3

> -3

> bye
